JSDemos/Demos/Charts/AutoCalculatedBarWidth/description.md

```diff
@@ -1 +1,3 @@
-When values for a&nbsp;series are not specified or&nbsp;are set to&nbsp;zero, bars that represent these values will not be&nbsp;visible within the chart. You can populate these empty region by&nbsp;extending neighboring bars via the **equalBarWidth** property set to _false_.
\ No newline at end of file
+When you have multiple [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/), values for some of their points can be unspecified or set to zero. Bars for these points are not displayed, but space for them remains reserved. This results in gaps.
```

Comment:
"When you have" is not a great way to start a sentence. Technically it's the chart that has series, not "you". And "have" is usually considered a weak and vague verb. 

```suggestion
This demo illustrates a common issue in multi-series charts. Individual [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/) may skip values for certain arguments or include zero values. If such cases arguments may display different number of bars. This results in uneven gaps between data. 
```